libs/api/domains/social-insurance/src/lib/dtos/paymentPlan.input.ts

```diff
@@ -0,0 +1,7 @@
+import { InputType, Field, Int } from '@nestjs/graphql'
+
+@InputType()
+export class SocialInsurancePaymentPlanInput {
```

Comment:
Inconsistent with the models so I would opt for the short class name and set the gql type name as argument
```suggestion
@InputType('SocialInsurancePaymentPlanInput')
export class PaymentPlanInput {
```

Reply:
will do